Plugins/DotNET/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 add_plugin(DotNET
     "DotNET.cpp"
-    "DotNETExports.cpp")
+    "DotNETExports.cpp")
```

Comment:
Should probably revert all changes to DotNET/ - ere, and DotNET/DotNET.i

Reply:
I'm still seeing DotNET/DotNET.i file and a change (removed newline) in DotNET/CMakeLists.txt.